core/templates/pages/exploration-player-page/learner-experience/tutor-card.component.ts

```diff
@@ -144,7 +144,6 @@ export class TutorCardComponent {
         }
       )
     );
-    this.updateDisplayedCard();
```

Comment:
Why is this no longer required?

Reply:
ngOnChanges takes care of this call.

I logged the number of calls made to updateDisplayCard function, I found unnecessary calls were made to to this function which caused flickering re rendering of view.

Reply:
Oh nice find, thanks for removing it then!